### DISCUSSION TITLE
Appimage version not opening urls in default web browser (XFCE)

### DISCUSSION THREAD

**Discussion by JNC92** [keepassxreboot/keepassxc#8760] [Q&A]
Hi,

With the version 2.7.4, clicking an url does nothing at all. Brave is set as my default browser, works fine with every other applications.
Running xubuntu 22.10 (hence XFCE).

Any hint?

**Comment by droidmonkey** [comment]
#8690

**Reply by JNC92** [reply]
Thanks for the speedy reply. But I am using 2.7.4 as stated in my post. Or have I missed something?

**Reply by JNC92** [reply]
For the record the output, will try to work on it:
/usr/bin/exo-open: symbol lookup error: /lib/x86_64-linux-gnu/libgobject-2.0.so.0: undefined symbol: g_uri_ref

**Reply by droidmonkey** [reply]
This isn't a keepassxc version problem, its a problem with appimage in general

**Reply by JNC92** [reply]
I'm lost: you say it's fixed in #8690 (Appimage too). Then it's a WONTFIX, correct?

**Reply by droidmonkey** [reply]
I linked the wrong one https://github.com/keepassxreboot/keepassxc/issues/8721

**Reply by JNC92** [reply]
OK, thanks. Will keep the flatpak version then.
